File: 백준/Bronze/1919. 애너그램 만들기/애너그램 만들기.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace MyCompiler {
    class Program {
        public static void Main(string[] args) {

            // 단어 입력
            string str1 = Console.ReadLine();
            string str2 = Console.ReadLine();

            // 소문자 카운트 값 저장할 배열
            int[] arr1 = new int[26];
            int[] arr2 = new int[26];

            // 소문자 개수 카운팅
            for(int i = 0 ; i < str1.Length; i++)
            {
                int index = str1[i] - 'a';
                arr1[index]++;
            }

            for(int i = 0 ; i < str2.Length; i++)
            {
                int index = str2[i] - 'a';
                arr2[index]++;
            }

            // 바꿔야 할 개수 체크
            int cnt = 0;
            
            for(int i = 0; i < arr1.Length; i++)
            {
                if(arr1[i] == arr2[i]) continue;

                // 첫번째 단어 소문자 개수가 더 많을 때
                if(arr1[i] > arr2[i]) cnt += (arr1[i] - arr2[i]);

                // 두번째 단어 소문자 개수가 더 많을 때
                else cnt += (arr2[i] - arr1[i]);
            }

            Console.Write(cnt);
            
        }
    }
}

File: 백준/Bronze/1919. 애너그램 만들기/애너그램 만들기.cc
#include <bits/stdc++.h>

using namespace std;

int main() {

    // 첫번째 단어
    string str1;

    // 두번째 단어
    string str2;

    // 단어 입력
    cin >> str1;
    cin >> str2;

    // 소문자 배열
    int arr1[26] = {0};
    int arr2[26] = {0};

    // str1 소문자 체크
    for(int i = 0; i < str1.size(); i++)
    {
        int index = str1[i] - 'a';

        arr1[index]++;
    }

    // str2 소문자 체크
    for(int i = 0; i < str2.size(); i++)
    {
        int index = str2[i] - 'a';

        arr2[index]++;
    }

    // 카운트 증가
    int cnt = 0;
    
    for(int i = 0; i < 26; i++)
    {
        // 같은 글자이면 패스
        if(arr1[i] == arr2[i]) continue;

        // 소문자 카운트가 더 많은 곳에서 뺀 값 총 카운트에 더하기
        if(arr1[i] > arr2[i]) cnt += (arr1[i] - arr2[i]);

        else if(arr2[i] > arr1[i]) cnt += (arr2[i] - arr1[i]);
    }

    cout << cnt;
}